.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+name: Setup Workload Identity 
+
+on:
+  workflow_call:
+    inputs:
+      skr-client-akv-endpoint:
+        description: "AKV endpoint for key release"
+        required: true
+        type: string
+      skr-client-maa-endpoint:
+        description: "Attestation endpoint"
+        required: true
+        default: "sharedeus2.eus2.test.attest.azure.net"
+        type: string
+      skr-client-kid:
+        description: "Key release key id"
+        required: true
+        default: "kafka-encryption-demo"
+        type: string
+      skr-kafka-demo-topic:
+        description: "Kafka topic"
+        required: true
+        default: "kafka-demo-topic"
+        type: string
+      cluster-name:
+        required: true
+        description: AKS Cluster Name With Randomly Generated Suffix
```

Comment:
Similar to the key, just have a randomly generated "run ID" for the workflow run and append it to a name: "kakfa-demo-cluster-randomnumber"

Reply:
ok I can do this.